src/components/plots/stackedAreaPlot.ts

```diff
@@ -101,10 +103,10 @@ export module Plots {
 
       var yAccessor = this.y().accessor;
       var xAccessor = this.x().accessor;
-      attrToProjector["y"] = (d: any, i: number, dataset: Dataset, m: StackedPlotMetadata) =>
-        this.y().scale.scale(+yAccessor(d, i, dataset, m) + m.offsets.get(xAccessor(d, i, dataset, m)));
-      attrToProjector["y0"] = (d: any, i: number, dataset: Dataset, m: StackedPlotMetadata) =>
-        this.y().scale.scale(m.offsets.get(xAccessor(d, i, dataset, m)));
+      attrToProjector["y"] = (d: any, i: number, dataset: Dataset, m: PlotMetadata) =>
+        this.y().scale.scale(+yAccessor(d, i, dataset, m) + this._stackOffsets.get(dataset).get(xAccessor(d, i, dataset, m)));
+      attrToProjector["y0"] = (d: any, i: number, dataset: Dataset, m: PlotMetadata) =>
```

Comment:
I guess PlotMetadata will disappear completely from this signature in one of your future issues?